lib/commands/element.js

```diff
@@ -9,6 +9,10 @@ function toBool (s) {
   return _.isString(s) ? (s.toLowerCase() === 'true') : !!s;
 }
 
+commands.active = async function active () {
+  return await this.uiautomator2.jwproxy.command('/element/active', 'GET', {});
```

Comment:
the last argument is redundant here since GET requests don't have bodies

Reply:
removed